source/world_builder/features/continental_plate.cc

```diff
@@ -295,6 +295,20 @@ namespace WorldBuilder
                         //std::cout << "vel=" << output[entry_in_output[i_property]] << ":" << output[entry_in_output[i_property]+1] << ":" << output[entry_in_output[i_property]+2] << std::endl;
                         break;
                       }
+                      case 7: // elevation
+                      {
+                        if(!std::isnan(output[entry_in_output[i_property]])){
```

Comment:
Are you sure this should not be:
```suggestion
                        if(std::isnan(output[entry_in_output[i_property]])){
```

Do you want to check here if the output is not a number in order to then set it to some number?

Reply:
That was a mistake. This is addressed in Commit [0599baa](https://github.com/GeodynamicWorldBuilder/WorldBuilder/pull/790/commits/0599baaa1902a24c10aec7df2cff1d15bc7b4252). Thank you!